### PR TITLE
[Fix] Fix open slice simulator

### PR DESCRIPTION
**Pull request by Heyrows**
## Description

- Small fix to initialise the tracker on the `_app` before the page component is rendered.

### CODE REVIEW

File: packages/slice-machine/components/Simulator/index.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ export default function Simulator() {
 
   useEffect(() => {
     // eslint-disable-next-line @typescript-eslint/no-floating-promises
```
Comment (Miniplop) [inline packages/slice-machine/components/Simulator/index.tsx]: so we can remove this right ?

File: packages/slice-machine/pages/_app.tsx [hunk 2]
```diff
@@ -78,6 +81,12 @@ function MyApp({ Component, pageProps }: AppContext & AppInitialProps) {
       },
     });
 
+    Tracker.get().initialize(
```
Comment (Miniplop) [inline packages/slice-machine/pages/_app.tsx:84]: nice

File: packages/slice-machine/components/Simulator/index.tsx (hunk 1 above)
Comment (MarcMcIntosh) [inline packages/slice-machine/components/Simulator/index.tsx]: is this rule still needed?

File: packages/slice-machine/pages/_app.tsx [hunk 3]
```diff
@@ -78,6 +81,12 @@ function MyApp({ Component, pageProps }: AppContext & AppInitialProps) {
...
+      process.env.NEXT_PUBLIC_SM_UI_SEGMENT_KEY ||
```
Comment (MarcMcIntosh) [inline packages/slice-machine/pages/_app.tsx:85]: Does this env var make it into the build?

File: packages/sm-commons/methods/actions.js [hunk 4]
```diff
@@ -39,39 +38,6 @@ function readConfig(p) {
   }
 }
 
-function readJsonPackage(p) {
```
Comment (MarcMcIntosh) [inline packages/sm-commons/methods/actions.js:42]: is this function not used any-where?

**Review by MarcMcIntosh — COMMENTED** [review]
A few questions, but it looks good

File: packages/sm-commons/methods/actions.js (hunk 4 above)
Comment (Heyrows) [reply packages/sm-commons/methods/actions.js:42]: no, that was an old start command apparently. It's coming from the cherry pick.

File: packages/slice-machine/components/Simulator/index.tsx (hunk 1 above)
Comment (Heyrows) [reply packages/slice-machine/components/Simulator/index.tsx]: apparently not :D I removed it

**Review by Miniplop — COMMENTED** [review]
LGTM

**Review by Miniplop — APPROVED** [review]

**Pull request merged**